packages/react-composites/src/composites/CallComposite/utils/Utils.ts

```diff
@@ -184,3 +186,70 @@ export const computeVariant = (
     return 'NO_STATE';
   }
 };
+
+/**
+ * Hook to override the avatar persona data of the users
+ * - Calls {@link AvatarPersonaDataCallback} on each render for provided `userIds`
+ * - Returns an array of the same length as `userIds`. Entries in the array may be undefined if there is no data to return.
+ * @private
+ */
+export const useCustomAvatarPersonaData = (
+  userIds: (string | undefined)[],
+  callBack?: AvatarPersonaDataCallback
```

Comment:
nitty nit: callback is one word so camel case doesn't apply here, just `callback`